#BE-4071 Testing Distribution Updates (#565)
appcircleio/appcircle-docusaurus: docs/distribute/create-or-select-a-distribution-profile.md

<Screenshot url='https://cdn.appcircle.io/docs/assets/image (153).png' />

### Publisher Information

The Info tab allows you to enter the publisher information for your distributed applications. 

You can submit your **Publisher Name**, **Contact Email**, **Privacy Policy URL**, and **Terms of Service URL**.

<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4071-info.png' />

Once you click the save button, the information you have provided will be displayed on the Testing Portal. 

> If the tester selects the user icon, they will be able to see the Publisher Information.

Reviewer: ```suggestion
When the tester selects the user icon, the Publisher Information will be displayed.
```

<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4071-info2.png' />

It will also display the Login Method for the Testing Distribution Profile. 

> In the example image, the profile has static authentication method, so it is displayed as Static Login. 

Reviewer: ```suggestion
In the example image, the profile uses a static authentication method, so it is displayed as Static Login.
```

> You can find out more about the login methods on [using authentication for distribution](/distribute/create-or-select-a-distribution-profile#using-authentication-for-distribution) section.

Reviewer: ```suggestion
You can find out more about the login methods in the [using authentication for distribution](/distribute/create-or-select-a-distribution-profile#using-authentication-for-distribution) section.
```
